pom.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
- Copyright (C) 2010 ZXing authors
+ Copyright (C) 2007—2020 ZXing authors
```

Comment:
The date doesn't matter to the copyright. I should have not bothered putting years in, actually, but have done it for all the files with year it was created. It's not worth changing.

Reply:
I’d say it’s either worth changing or worth removing, since these copyright notices are used by downstreams to keep track of copyrights and licenses.

Reply:
How does the date matter?